pages/Documents/DeveloperTools/LivePersonFunctions/DevelopingwithFaaS/events-templates.md

```diff
@@ -22,19 +22,19 @@ The amount of lambdas connected to a specific event is restricted and can be see
 
 Currently the following events are supported:
 
-|Event Name|Event Description|Event Source|Lambda Limit|
-|--- |--- |--- |--- |
+|Event Name|Event Description|Event Source|Lambda Limit|Supports Skills|
+|--- |----- |--- |--- |--- |
```

Comment:
Do the `-` determine the total space allocated for rendering? If so we should actually greatly reduce the ticks for `Lambda Limit` & `Supports Skills`, as the content here is very small. Else if it's a typo we should make sure `---` is used consistently 

```suggestion
|--- |--- |--- |--- |--- |
```